### ISSUE TITLE
Exception in EditInlineMessageTextAsync

### ISSUE THREAD

**Issue by samshak110** [MrRoundRobin/telegram.bot#325]
Hi 
i use EditInlineMessageTextAsync to edit the inline message from user. 
this function works correctly but at the end of execute send exception that say
"Error converting value True to type 'Telegram.Bot.Types.Message'."

i think exeption throw when try to convert Boolean response to Task<Message> in EditInlineMessageTextAsync method.

Thanks

**Comment by rgbweb** [comment]
This is true. It seems like Telegram is always returning a _true_ instead of a _Message_, when an inline message is edited. 

See also in documentation: [https://core.telegram.org/bots/api#editmessagetext](https://core.telegram.org/bots/api#editmessagetext)
> ### editMessageText
> Use this method to edit text and game messages sent by the bot or via the bot (for inline bots). On success, if edited message is sent by the bot, the edited Message is returned, otherwise _True_ is returned.

The implementation is incorrect. Same for _editMessageCaption_ and _editMessageReplyMarkup_.
